doc/releases/changelog-dev.md

```diff
@@ -59,44 +56,102 @@
   
   mol = qml.qchem.Molecule(symbols, geometry, basis_name='sto-3g')
   core, one, two = qml.qchem.electron_integrals(mol)()
-  algo = DoubleFactorization(one, two)
   
-  # print the number of non-Clifford gates and logical qubits
-  print(algo.gates, algo.qubits)
+  algo = DoubleFactorization(one, two)
   ```
 
   ```pycon
+  >>> print(algo.gates, algo.qubits)
   103969925, 290
   ```
 
-  The methods of the `FirstQuantization` and the `DoubleFactorization` classes can be also accessed
-  individually. For instance, the logical qubits can be computed by providing the inputs needed for
-  this estimation without initiating the class. 
+  The methods of the `FirstQuantization` and the `DoubleFactorization` classes 
+  can be also accessed individually without initiating the class:
+
+  - The number of logical qubits with `qubit_cost`:
 
   ```python
   n = 100000
   eta = 156
   omega = 169.69608
   error = 0.01
-  qml.resource.FirstQuantization.qubit_cost(n, eta, omega, error)
   ```
   
   ```pycon
+  >>> qml.resource.FirstQuantization.qubit_cost(n, eta, omega, error)
   4377
   ```
 
-  In addition to the number of non-Clifford gates and logical qubits, some other quantities such as
-  the 1-norm of the Hamiltonian and double factorization of the second-quantized Hamiltonian can be
-  obtained either by initiating the classes or by directly calling the functions.
+  - The number of non-Clifford gates with `gate_cost` (TODO: check this):
+
+   ```python
+    n = 100000
+    eta = 156
+    omega = 169.69608
+    error = 0.01
+    ```
+    
+    ```pycon
+    >>> qml.resource.FirstQuantization.gate_cost(n, eta, omega, error)
+    TODO
+    ``` 
```

Comment:
Just wrote this here as a placeholder. Does this feature exist?

Reply:
This worked! Will push soon. I had trouble building 0.25 from the repo for some reason yesterday 🤔. I fixed it just as the day ended and am now able to test things. All good!